zetasql/fuzzing/zetasql_expression_grammar.proto

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto2";
+
+package zetasql_expression_grammar;
+
+message Expression {
+    oneof expr_oneof {
+        LiteralExpr literal = 1;
+        CompoundExpr expr = 2;
+    }
+}
+
+message LiteralExpr {
+    enum SpecialValue {
+        V_NULL = 0; // variable 'NULL' will be translated to NULL in C, causing failure
+    }
+    oneof literal_oneof {
+        SpecialValue special_literal = 1;
+        bool bool_literal = 2;
+        string string_literal = 3;
+        bytes bytes_literal = 4;
+        IntegerLiteral integer_literal = 5;
+        NumericLiteral numeric_literal = 6;
+    }
+}
+
+message IntegerLiteral {
+    oneof integer_oneof {
+        int32 int32_literal = 1;
+        uint32 uint32_literal = 2;
+        int64 int64_literal = 3;
+        uint64 uint64_literal = 4;
+    }
+}
+
+// TODO: Support high precision value representation
+message NumericLiteral {
+    required bytes value = 1;
+}
+
+message CompoundExpr {
+    oneof compound_oneof {
+        AdditiveOperation additive = 1;
```

Comment:
Out of curiousity-- is there a reason we're separating addition/subtraction and multiplication/division?  Since they're all binary operations, this is a little odd.

Reply:
I guess we can do that too. I was completely mirroring the definition used by the Bison parser

Reply:
So the reason they'd be separated in the Bison parser is to allow you to easily preserve order of operations in the parse tree.  Since we don't need to do that, it's probably cleaner to consolidate them.  :)